pages/linux/apt-get.md

```diff
@@ -14,10 +14,14 @@
 
 `apt-get remove {{package}}`
 
-- Upgrade installed packages to newest available versions:
+- Upgrade all installed packages to newest available versions:
 
 `apt-get upgrade`
 
+- Upgrade a single package to the newest available version:
+
+`apt-get install --only-upgrade {{package}}`
```

Comment:
Umm .. you don't need to pass the `--only-upgrade` flag. By default, if the package is installed and you do `apt-get install`, it will upgrade it.

Reply:
I believe that `--only-upgrade` is precisely to ensure that the package isn't installed if it isn't already present in the machine. Can you confirm? But I can agree that it may be a niche case. What do you guys think, @agorf, @fuerbringer, @lord63, @zdroid?

Reply:
> Can you confirm?

Yes, that's right. But almost always, you would want to actually _install_ the package if you don't have it. So, its very much a corner case, where you want to upgrade a package, but you don't want to accidentally install it if it doesn't exist.

Reply:
True, trying to install a missing package with `--only-upgrade` skips the package installation in my system. But this sounds like an edge case to me: "do not install this package but only upgrade it if it's already installed". You'll probably already know if a package is installed or not. Whereas `apt-get install` always works like how you'd expect ("install this package if missing or upgrade it if present - either way I want it at the latest version").

Reply:
Fair enough. Should we then, instead of introducing a new example, change the description of the existing `apt-get install {{package}}` example to "Install a new package or update it to the latest available version"?

I'm a bit wary of mixing concepts / intentions here, which is why I initially added this next to the "upgrade all" option, as it semantically makes more sense there.

Reply:
+1 for a single example with the detailed description you mention.

Reply:
> Should we then, instead of introducing a new example, change the description of the existing apt-get install {{package}} example to "Install a new package or update it to the latest available version"?

By all means, yes. :)

Reply:
Ok then. Can you confirm that this is true for both `apt` and `apt-get`?

Reply:
Yes.